cylc/uiserver/data_store_mgr.py

```diff
@@ -351,7 +407,6 @@ def _reconcile_update(self, topic, delta, w_id):
                         'pb_data_elements',
```

Comment:
Do we need to use run_coroutine_threadsafe` any more (line 404)?

Reply:
No, I would think not.

Reply:
Added comment but haven't removed right now as this would require a little more async wrangling.